frontend/public/components/storage-class-form.tsx

```diff
@@ -1094,6 +1094,7 @@ class StorageClassFormWithTranslation extends React.Component<
                 onClick={this.createStorageClass}
                 type="submit"
                 variant="primary"
+                data-test="create-action"
```

Comment:
Did you miss this? We could use instead `button[type="submit"]` where it is required?